packages/dx-react-grid/docs/components/table-header-cell.md

```diff
@@ -0,0 +1,33 @@
+# TableHeaderCell Component Reference
+
+A component that represents the default table header cell.
+
+## User Reference
+
+### Properties
+
+Field | Type | Description
+------|------|------------
+column | [Column](#column) | A column object.
+tableColumn | [TableColumn](../reference/table-view.md#table-column) | Table column.
+allowSorting | boolean | If true, an end-user can change sorting by a column.
+allowResizing | boolean | If true, an end-user can resize a column.
+sortingDirection? | 'asc' &#124; 'desc' | Specifies the column's sorting order.
+changeSortingDirection | ({ keepOther: boolean, cancel: boolean }) | Changes column sorting direction. Keeps the existing sorting if `keepOther` is set to `true`. Cancels sorting by the current column if `cancel` is set to true.
```

Comment:
The 'on' prefix absence looks too bad in the component reference

Reply:
Not sure, we understand this comment)